lib/game/game.dart

```diff
@@ -366,10 +391,12 @@ class GameRenderBox extends RenderBox with WidgetsBindingObserver {
   }
 
   void _scheduleTick() {
+    _running = true;
     _frameCallbackId = SchedulerBinding.instance.scheduleFrameCallback(_tick);
   }
 
   void _unscheduleTick() {
+    _running = false;
     SchedulerBinding.instance.cancelFrameCallbackWithId(_frameCallbackId);
```

Comment:
could _frameCallbackId be null?

Reply:
I don't think so, I haven't changed this, and this seems to be working so far...